changeset:   12916:6f35dc12506a
branch:      wip/dueno/typo-fix
tag:         tip
parent:      12913:f2a9e4d85b64
user:        Daiki Ueno <dueno@redhat.com>
date:        Tue Nov 29 14:18:08 2016 +0100
files:       tests/ssl/ssl.sh
description:
Use correct shell conditional for NSS_DISABLE_LIBPKIX check


diff --git a/tests/ssl/ssl.sh b/tests/ssl/ssl.sh
--- a/tests/ssl/ssl.sh
+++ b/tests/ssl/ssl.sh
@@ -1006,7 +1006,7 @@ ssl_run()
     do
         case "${SSL_RUN}" in
         "stapling")
-            if [ -nz "$NSS_DISABLE_LIBPKIX" ]; then
+            if [ -z "$NSS_DISABLE_LIBPKIX" ]; then
               ssl_stapling
             fi
             ;;

